src/tile-plotting.jl

```diff
@@ -211,6 +211,11 @@ function create_tileplot!(
         colorrange=data.elevation_range,
         config.attributes...
     )
+    # so that things get plotted on top of this
+    # note that all translation / scaling is done by the arguments
+    # not by modifying the plot's transformation as was done in previous Tyler v0.1.5 and below
+    # so this should hold indefinitely
+    translate!(p, 0, 0, -10) 
```

Comment:
This seems to be in the `ElevationData` dispatch ...

Reply:
doh, yeah that should be in the imagedata dispatch

Reply:
Even in the right spot this approach doesnt seem to work, during zoom tiles cover e.g.  a scatter plot anyway. But at least the scatter does come back after zooming

Reply:
And seems sometimes they dont come back after zooming either

Reply:
We need to turn down the depth_shift to something like 'n / 100000f0' and then it works on my branch. so some progress!

Reply:
Eps is for values at zero. It's obvious what you're doing but the scale will need to be much larger

Reply:
The range is always (-1, 1) though, and eps with a number is actually the correct nextfloat epsilon:
```julia
julia> eps(1f0)
1.1920929f-7

julia> eps(10000f0)
0.0009765625f0
```

But either way works

Reply:
The numbers are between 0 and 100 before scaling

Reply:
You can review the PR ad edit the code inline if you really want to change it ;)